### DISCUSSION TITLE
Time Series: variable point size

### DISCUSSION THREAD

**Discussion by yulipumal** [grafana/grafana#66667] [Dashboards, Panels & Visualizations]
Hi,
currently, the point size of the graph can only be set statically.
Is there the plan to implement the point size dynamically, depending on a query column?
This is possible for example in XYChart.

Thanks for the answer

**Comment by leeoniya** [comment]
no current plans. it's an unusual feature for a line chart. XY has this because the points are hoverable and allows for making bubble plots.

**Comment by yulipumal** [comment]
Thanks for the answer @leeoniya. 

I see a useful feature here, if you have another information that you want to encode in your point plot. E.g. frequency of values.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82737. Feel free to continue the discussion around this there. Thank you!
